specification/azurefleet/AzureFleet.Management/fleet.tsp

```diff
@@ -17,7 +17,7 @@ model Fleet is TrackedResource<FleetProperties> {
   @doc("The name of the Compute Fleet")
   @key("fleetName")
   @segment("fleets")
-  @pattern("^[^_\\W][\\w-._]{0,79}(?<![-.])$")
+  @pattern("^[^_\\W][\\w\\-._]{0,79}(?<![-.])$")
```

Comment:
Please update an example with Fleet name which validates this new pattern as well.